README.md

```diff
@@ -13,16 +13,26 @@ issues, mailing lists, conferences, etc.
 
 For more specific topics, try a SIG.
 
-## SIGs
+## Governance
 
-Kubernetes is a set of subprojects, each shepherded by a Special Interest Group (SIG).
+Kubernetes has three types of groups that are officially supported:
 
-A first step to contributing is to pick from the [list of kubernetes SIGs](sig-list.md).
+* **Committees** are named sets of people that are chartered to take on sensitive topics.
+  This group is encouraged to be as open as possible while achieving its mission.
```

Comment:
Is it worth mentioning that while they are encouraged to be open, by the nature of the topics they discuss committees are permitted to have private communications as well?

Reply:
> Is it worth mentioning that while they are encouraged to be open, by the nature of the topics they discuss committees are permitted to have private communications as well?

I think "as possible" already suggests this, but more explicit wording would be fine too.